R/opq.R

```diff
@@ -245,13 +264,11 @@ add_osm_feature <- function (opq,
         stop ("key must be provided")
     }
 
-    if (is.null (bbox) & is.null (opq$bbox)) {
+    if (is.null (bbox) && is.null (opq$bbox)) {
```

Comment:
Thanks for catching those!

Reply:
You are welcome. I spend some time reading the code :)